### ISSUE TITLE
[BE] Create app page info GraphQL query and mutation

### ISSUE THREAD

**Issue by ronaldheft-usds** [newjersey/dhs-oneapp#303]
Update the application type built in #297 to support the following GraphQL mutations:

* `OA_PKG_APP.SP_SELECT_APP_PAGE_INFO`
* `OA_PKG_APP.SP_UPDATE_APP_PAGE_INFO`
* `OA_PKG_GEN.SP_GET_LAST_PAGE_NUMBER`
* `OA_PKG_GEN.SP_UPDATE_LAST_PAGE_NUMBER`

These stored procedures may not be needed for the new version of OneApp. If that's the case, we need to test that an application created with the new version of OneApp will be backward compatible without these stored procedures. If not, we may have to insert dummy data into the DB to ensure compatibility.

Acceptance criteria:

* The application query calls the `_SELECT_` stored procedure.
* The application mutation calls the `_UPDATE_` stored procedure.
* The GraphQL type has validations that enforce the rules defined in the legacy ASP pages.

**Comment by mayaisrani** [comment]
from digging around https://github.com/newjersey/dhs-oneapp-legacy/blob/080a57380e9dedd834d2b6de141d7a031825159e/NJOneAppAuto/Forms/Application/Submit.aspx.cs, looks like this class might be deprecated at this point. looking further

**Comment by ronaldheft-usds** [comment]
Yeah, I think we can drop this task from the requirements for the MVP. It makes sense if we want to be fully backward compatible (so that someone could start their app in the new version and finish in the old version), but when I did a review of the submit stored procedure a month or so back, I didn't find any evidence that that any of the app page info was used in the actual application submission. It's all state data used for the legacy OneApp to know what page in the application to return the user to when they log back in.

**Comment by mayaisrani** [comment]
sounds good, thanks Ron. will leave for now
